api/http/router.go

```diff
@@ -69,8 +75,12 @@ func NewRouter(app app.App) http.Handler {
 	intrnlGrp.POST(URITenantDevices, intrnlAPI.ProvisionDevice)
 	intrnlGrp.DELETE(URITenantDevice, intrnlAPI.DecommissionDevice)
 
-	// mgmtAPI := NewManagementAPI(app)
+	mgmtAPI := NewManagementAPI(app)
 	mgmtGrp := router.Group(URIManagement)
+
+	mgmtGrp.GET(URIConfiguration, mgmtAPI.ConfigurationGet)
+	mgmtGrp.PUT(URIConfiguration, mgmtAPI.ConfigurationSet)
+
 	// identity middleware for collecting JWT claims into request Context.
 	mgmtGrp.Use(identity.Middleware())
```

Comment:
> @merlin-northern This is why it didn't work. The order matters - `*.Use` must come first, otherwise the middleware is not pushed on the handler stack.

ha! I knew it! thanks!